### ISSUE TITLE
FirebaseAuthImplementation does not have an Initialize method

### ISSUE THREAD

**Issue by cagriy** [TobiasBuchholz/Plugin.Firebase#233]
What am I missing here?

I'd like to use email authentication, the basic setup is done, now reading this page:
https://github.com/TobiasBuchholz/Plugin.Firebase/blob/master/docs/auth.md
It instructs adding `FirebaseAuthImplementation.Initialize();` after `CrossFirebase.Initialize()`

However,
https://github.com/TobiasBuchholz/Plugin.Firebase/blob/master/src/Auth/Platforms/iOS/FirebaseAuthImplementation.cs
does not implement an `Initialize` method.

Looked into Sample application as well, it seems like it only has samples for Facebook and Google auth, but not email.

What do I need to do to setup and use email auth correctly ?

Many thanks

**Comment by TobiasBuchholz** [comment]
Yes, you are right, since extracting Google Auth into it's own [Plugin.Firebase.Auth.Google](https://www.nuget.org/packages/Plugin.Firebase.Auth.Google/) nuget package the `FirebaseAuthImplementation.Initialize()` method was replaced by `FirebaseAuthGoogleImplementation.Initialize()`. I've just updated the readme accordingly, thanks for the hint! 

For using email auth there are no additional steps needed, so it actually should just work.
